_i18n/ar/resources/moneropedia/networkupgrade.md

```diff
@@ -0,0 +1,8 @@
+---
+terms: ["networkupgrade"]
+summary: Network Upgrades (hard forks) are non-contentious and no new coin will be created. We have multiple blog posts we could link to.
+---
+
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+Network Upgrades (hard forks) are non-contentious and no new coin will be created. We have multiple blog posts we could link to.
```

Comment:
This is a bit short. If we explain what network upgrades are we should be a bit more verbose and if we mention a blog post would be good to link to it.